aws-observability/templates/sumologic_observability.master.template.yaml

```diff
@@ -297,7 +297,7 @@ Parameters:
     Type: String
     Description: "Required when already collecting Lambda CloudWatch logs in Sumo Logic. Provide the existing Sumo Logic Lambda CloudWatch Source API URL. Account, region and namespace Fields will be added to the Source. For Source API URL, visit https://help.sumologic.com/03Send-Data/Sources/03Use-JSON-to-Configure-Sources/Local-Configuration-File-Management/View-or-Download-Source-JSON-Configuration"
     Default: ""
-  Section7cAutoSubscribeLogGroupsLambdaOptions:
+  Section7cAutoSubscribeLogGroupsOptions:
```

Comment:
Before pushing code change for variable names I would recommend to perform update testing from v2.7.0 to v2.8.0.